PROJECTS.rst

```diff
@@ -46,6 +46,7 @@ Full screen applications:
 - `pypager <https://github.com/prompt-toolkit/pypager>`_: A $PAGER in pure Python (like "less").
 - `kubeterminal <https://github.com/samisalkosuo/kubeterminal>`_: Kubectl helper tool.
 - `pydoro <https://github.com/JaDogg/pydoro>`_: Pomodoro timer.
+- `sanctuary-zero <github.com/t0xic0der/sanctuary-zero>`_: A secure chatroom with zero logging and total transience
```

Comment:
This is missing the `https://` prefix. It is a broken link right now.

Reply:
Right, I'll fix that.

Reply:
Fixed here: https://github.com/prompt-toolkit/python-prompt-toolkit/commit/551a5b8fb4901ea24dc334da707e84bcf0ddc6d0 Thanks for pointing out!